tests/ci_visibility/test_ci_visibility.py

```diff
@@ -102,6 +115,32 @@ def test_ci_visibility_service_enable():
             CIVisibility.disable()
 
 
+def test_ci_visibility_service_enable_without_service():
```

Comment:
In what sense is this "without service"?

Reply:
That `CIVisibility.enable()` is called without `service='test-service'`.